### PR TITLE
Sphinx doc chapter 18

### PR DESCRIPTION
**Pull request by maximedenes**

### CODE REVIEW

File: doc/sphinx/language/gallina-extensions.rst [hunk 1]
```diff
@@ -2189,12 +2189,6 @@ an inductive type or any constant with a type of the form
 Then the user is able to apply an object that is not a function, but
 can be coerced to a function, and more generally to consider that a
 term of type ``A`` is of type ``B`` provided that there is a declared coercion
-between ``A`` and ``B``. The main command is
-
-.. cmd:: Coercion @qualid : @class >-> @class.
-
-which declares the construction denoted by qualid as a coercion
-between the two given classes.
```
Comment (silene) [inline doc/sphinx/language/gallina-extensions.rst]: What is that change?

Comment (maximedenes) [reply doc/sphinx/language/gallina-extensions.rst]: This command had two entries in the table, in two chapters. It is not supported by our new infrastructure, but anyway this entry was mainly pointing to the other one, so I simply removed it.

Comment (silene) [reply doc/sphinx/language/gallina-extensions.rst]: You removed too much (or not enough).

Comment (maximedenes) [reply doc/sphinx/language/gallina-extensions.rst]: Ouch! I guess I haven't slept enough. Good catch :)

**Comment by Zimmi48** [comment]
@maximedenes Can you resolve the conflicts by rebasing?

**Comment by Zimmi48** [comment]
There are three FIXME in this chapter, probably meant to be addressed before merging. Otherwise, LGTM.

**Comment by maximedenes** [comment]
The `FIXMEs` are things that should be moved to other chapters, so I'd prefer to postpone it, and address them after the integration has a bit stabilized.

**Comment by maximedenes** [comment]
@silene 's remarks addressed, rebased.

**Comment by Zimmi48** [comment]
Don't the FIXME risk to appear in the rendered text?

**Comment by maximedenes** [comment]
Argh! I hadn't realized they were not comments...

**Comment by maximedenes** [comment]
Pipeline here: https://gitlab.com/maximedenes/coq/pipelines/19693233

Should be ready to go once CI says ok.

**Comment by maximedenes** [comment]
@silene Pipeline succeeded modulo one spurious failure. Can you merge?

**Comment by maximedenes** [comment]
@silene ping

**Comment by SkySkimmer** [comment]
Circle is "Too long with no output (exceeded 10m0s)" during iris-coq's coqchk with 19min total run time.
Gitlab is "execution took longer than 1h0m0s" compiling lambdaRust.

**Comment by Zimmi48** [comment]
Is there an equivalent of `travis_wait` for CircleCI?

**Comment by ejgallego** [comment]
BTW I have my own pipelines set to 90 minutes timeout. The "large" server I set up indeed is performing way worse than anticipated [and it seems that runs from other people occasionally kick-in making docker die], so yeah, we should look for a nice server [or use AWS, but for that we want to optimize the docker image first].

**Comment by maximedenes** [comment]
@silene I think we should merge these PRs quickly, because some other PRs need to add documentation, but they can't see the Sphinx version...

**Comment by Zimmi48** [comment]
@silene I'm going to take care of merging these Sphinx PRs. I hope you don't mind but it seems that you are away from keyboard ATM and it's a bit of a special case given that I reviewed these PRs.
Also, we should consider having a third code owner, for cases where PRs are submitted by a code owner and the other one is on vacation.

**Pull request merged**